std/StandardToken.sol

```diff
@@ -0,0 +1,43 @@
+import "./Token.sol";
+
+contract StandardToken is Token {
+	uint256 public totalSupply;
+	mapping (address => uint256) public balanceOf;
+	mapping (address =>
+		mapping (address => uint256)) public allowance;
+
+	function StandardToken(address _initialOwner, uint256 _supply) {
+		totalSupply = _supply;
+		balanceOf[_initialOwner] = _supply;
+	}
+
+	function transfer(address _to, uint256 _value) returns (bool success) {
+		if (balanceOf[msg.sender] >= _value) {
```

Comment:
To be safe (the initial supply can be specified in the constructor), we should also do
`balanceOf[_to] + _value >= balanceOf[_to]`.